src/uu/install/src/install.rs

```diff
@@ -626,6 +628,62 @@ fn copy_files_into_dir(files: &[PathBuf], target_dir: &Path, b: &Behavior) -> UR
     Ok(())
 }
 
+/// Handle incomplete user/group parings for chown.
+///
+/// Returns a Result type with the Err variant containing the error message.
+///
+/// # Parameters
+///
+/// _path_ must exist as.
+///
+/// # Errors
+///
+/// If the owner or group are invalid or copy system call fails, we print a verbose error and
+/// return an empty error value.
+///
+fn chown_optional_user_group(path: &Path, b: &Behavior) -> UResult<()> {
+    let owner = if !b.owner.is_empty() {
+        match usr2uid(&b.owner) {
+            Ok(u) => Some(u),
+            _ => return Err(InstallError::NoSuchUser(b.owner.clone()).into()),
```

Comment:
Can we create tests for the lines that codecov reports are untested?

Reply:
Tests added and error strings corrected to match GNU coreutils for invalid users.

Would like to test the operation, but unclear how to `chown` as an unprivileged user on the build server.  Probably not feasible?

Reply:
Yeah, I think you're right that's not possible at the moment.